openedx/features/caliper_tracking/tests/current/edx.bookmark.removed.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file is not related to this event. Please remove it.

Reply:
Done already.